src/Esfa.Vacancy.Register.Api/Controllers/SearchApprenticeshipVacanciesController.cs

```diff
@@ -17,18 +18,80 @@ public SearchApprenticeshipVacanciesController(SearchApprenticeshipVacanciesOrch
         }
 
         /// <summary>
-        /// The Vacancy Summary service provides system integrators with the ability to download open apprenticeships 
-        /// from the Recruit an apprentice system. This service allows various parameters to allow the system 
-        /// integrators to select the most relevant apprenticeships and/or request up-to date details after a given date.
+        /// The apprenticeship search operation retrieves live apprenticeship vacancies based on search criteria specified 
+        /// in the request parameters. 
+        /// 
+        /// Search criteria can be used to:
+        /// 
+        /// - Search by framework code(s)
+        /// - Search by standard code(s)
+        /// - Search by framework or standard code(s)
+        /// - Search for recently posted vacancies
+        /// 
+        /// #### Data paging ####
+        /// 
+        /// Search results are returned in pages of data. 
+        /// If not specified then the default page size is 100 vacancies. 
+        /// If the search yields more data than can be included in a single page then additional pages can be requested by 
+        /// specifying a specific page number in the request. eg. pageNumber=2
+        /// 
+        /// #### Examples ####
+        /// 
+        /// To search for vacancies with standard code 94:
+        /// 
+        /// ```
+        /// /apprenticeship/search?standardCodes=94
+        /// ```
+        /// 
+        /// Multiple standard codes can be specified by using a comma delimited of standard codes. 
+        /// To search for vacancies with either standard code 94 or 95:
+        /// 
+        /// ```
+        /// /apprenticeship/search?standardCodes=94,95
+        /// ```
+        /// 
+        /// To search for vacancies that went live today since 2 days ago:
+        /// 
+        /// ```
+        /// /apprenticeship/search?postedInLastNumberOfDays=2
+        /// ```
+        /// 
+        /// To search for vacancies that went live today (0 days ago):
+        /// 
+        /// ```
+        /// /apprenticeship/search?postedInLastNumberOfDays=0
```

Comment:
That's not how it is implemented, zero is considered as no filter.

Reply:
@dev4valtech as discussed, can you update the implementation so that users can search for "today's vacancies"